src/types/CoverageIntegrationRun.ts

```diff
@@ -2,22 +2,24 @@ import type { CoverageIntegrationRef } from './CoverageIntegration';
 
 type Iso8601DateTime = string;
 
-export enum Status {
-    DONE = 'done',
-    ERROR = 'error',
-    RUNNING = 'running',
-}
-
 export interface CoverageIntegrationRun {
     id: number;
     uuid: string;
     created_at: Iso8601DateTime;
     updated_at: Iso8601DateTime;
-    status: Status;
+    status: CoverageIntegrationRun.Status;
     /**
      * Following properties will be `null` when status = 'running'
      */
     stats_processed: number | null;
     stats_added: number | null;
     integration: CoverageIntegrationRef;
 }
+
+export namespace CoverageIntegrationRun {
+    export enum Status {
```

Comment:
**Before this change:**

The consumer had to import the type from `@prezly/sdk` like this:

```tsx
import { Status } from "@prezly/sdk";

// Usage
Status

// It's not clear what Status refers to. ✖️
// There's a great chance some other entity will have `Status` too. e.g. a story's `Status` ✖️
```

**After this change:**

```tsx
import { CoverageIntegrationRun } from "@prezly/sdk";

// Usage
CoverageIntegrationRun.Status

// It's clear what status refers to. A coverage integration run ✔️
// No need to worry about a naming conflict with another entity exposing status anymore. ✔️

Reply:
Yes, that `Status` worried me many times. Good job refactoring it!